Formula/usbutils.rb

```diff
@@ -0,0 +1,43 @@
+class Usbutils < Formula
+  desc "Provides 'lsusb' tool for getting detailed info about USB devices"
```

Comment:
```suggestion
  desc "Get detailed info about USB devices"
```

Reply:
or "List"

Reply:
Yeah, this is also an option
```suggestion
  desc "List detailed info about USB devices"
```